### DISCUSSION TITLE
Mobile version of GitHub.com doesn't allow us to see who reacted to a comment

### DISCUSSION THREAD

**Discussion by 0xdevalias** [community/community#4196] [Mobile] [answered]
I originally shared this in a tweet ([ref](https://twitter.com/_devalias/status/1398818459845226498?s=21)) but figured I would officially raise it here too:

> As far as ‘papercut problems’ go, it would be so awesome if Github allowed a ‘long tap’ on issue reaction emoji’s to display the users who reacted. Don’t think it’s possible to get this info on mobile web view at all currently.

The desired outcome would be that on mobile, by tapping and holding (aka: 'long tap') on the 'emoji reactions' section on a comment, I would be able to see the 'onhover' tooltip that I can currently get on desktop, and see who reacted:

![image](https://user-images.githubusercontent.com/753891/122306210-6477c600-cf4b-11eb-94c9-875deb0d1778.png)

---

Note that I also raised this directly with support in 🔒 https://support.github.com/ticket/personal/0/1171585 and received the following reply:

> Thank you for sharing your thoughts, comments, and ideas with us! Your ticket was automatically routed to our Developer Support team but our Product team is better suited to help you out. We have forwarded your ticket to the Product team who reads and evaluates all feedback, however we cannot guarantee a response to every submission.
> 
> Please note that this means the Developer Support team will no longer be able to see changes to the ticket, so if you have a follow up that requires further support, please create a new ticket. Thank you again for your feedback!

**Comment by alcere** [comment, marked as answer]
Hi! Thanks for your feedback! Seeing who and how they reacted on an issue is possible in the mobile app, but the behaviour differs slightly from the web version; you only need to press and hold on to any reaction left in the comment.

**Reply by 0xdevalias** [reply]
It would be great if this was possible in the mobile web UI. I still find email -> web UI a more convenient flow most of the time than using the app, as I’m usually clicking through from the link in the email notification, which doesn’t seem to open in the app.

**Reply by TPS** [reply]
> I’m usually clicking through from the link in the email notification, which doesn’t seem to open in the app.

It depends on _which_ version of the mobile app:  in my experience, about every 3rd GitHub app update seems to lose the ability to open links (especially from GMail, which further messes w/ external links), but is resolved on the following update.  I agree it's weird _**&**_ annoying.

**Reply by alcere** [reply]
> It would be great if this was possible in the mobile web UI

This should be possible to implement, I will create an internal issue to make sure we evaluate the option.

> I’m usually clicking through from the link in the email notification, which doesn’t seem to open in the app.

We are sorry about the inconvenience. We are aware that the deep link experience is sometimes not working as expected :(. The good news is that we are currently working on improving the experience; please keep tuned for changes. Thanks for your patience!

**Reply by alcere** [reply]
Can you share an example of a link that doesn't open the app, please? That would be very useful for us.

**Reply by 0xdevalias** [reply]
This, from the ‘view on GitHub’ at the bottom of email notifications: https://github.com/github/feedback/discussions/4196#discussioncomment-896768

Same story regardless of whether it’s a discussion, PR or issue.

Using the Gmail app on iOS, so unsure if it’s related to that.

**Reply by jsoref** [reply]
Fwiw, I've been using GitHub Mobile for a few months and I'm pretty sure I didn't discover this feature until I read this discussion (I wasn't looking for it, in looking into some other behavior).

The behavior isn't wrong, but I'm pretty sure it doesn't fit the rest of the behaviors I've discovered in the app and thus it doesn't feel discoverable. – It might be worth thinking about a way to nudge users to discover the feature.

**Reply by ws909** [reply]
This has not been possible in browsers on mobile phones, however, up until recently, a workaround was to tap the reaction given by somebody else, which would leave that same reaction from myself, and also bring up the list of the past reacters. Then I’d tap it again to remove my reaction. This no longer works. The list is not shown when reacting.

**Reply by Number-3434** [reply]
Will this be implemented on the web soon?

**Reply by HalfWhitt** [reply]
Still wishing this were implemented on web mobile. It's one of those things that's not a *huge* deal, but sometimes it is actually important. Knowing who reacted what is a piece of context.

**Reply by markcellus** [reply]
Tapping and holding on the reaction as mentioned in this https://github.com/orgs/community/discussions/4196#discussioncomment-881725 seems to work for me. Does that not show you who has reacted?

**Reply by HalfWhitt** [reply]
Huh. No, it doesn't. That's what it intuitively seemed like it should do, so I've been trying it. Then Googling it led me here.

**Reply by HalfWhitt** [reply]
Wait, that comment says you can do that in the mobile app. I'm talking about in a web browser.

@alcere do you know the status of that [internal ticket you mentioned](https://github.com/orgs/community/discussions/4196#discussioncomment-896748)?

**Reply by markcellus** [reply]
I'm using a mobile web browser right now and it works lol

**Reply by HalfWhitt** [reply]
Well then what the heck. 🤔

Guess I'd better file a bug report then. Not working for me in either Chrome or Safari on iOS.

**Reply by markcellus** [reply]
Yeah may be ios because im on Brave on Android. Also works on Chrome Android.

![Screenshot_20241113-170259](https://github.com/user-attachments/assets/e3252619-3b18-4426-a47c-5a3e36e41f04)

**Reply by sbrannen** [reply]
Still does not work in Safari or Chrome on iOS.

A "long press" simply highlights the up-voting "ellipse".

**Reply by HalfWhitt** [reply]
> This has not been possible in browsers on mobile phones, however, up until recently, a workaround was to tap the reaction given by somebody else, which would leave that same reaction from myself, and also bring up the list of the past reacters. Then I’d tap it again to remove my reaction. This no longer works. The list is not shown when reacting.

*This* seems to have started working again, at least.

**Comment by HalfWhitt** [comment]
This question about the mobile *website* is four years old, the accepted answer is about the mobile *app* instead, and this feature still isn't available in web browsers (at least Firefox and Safari) on iOS.
